tests/js/app/widgets/testFormattableLabel.js

```diff
@@ -53,6 +53,6 @@ describe('Formattable Label', function () {
         Utils.update_gui();
         label.use_markup = true;
         label.label = "Custer&apos;s Last Stand";
-        expect(label.label).toBe("CUSTER'S LAST STAND");
```

Comment:
The only question I have is how this worked before? When were we converting these?